app/routes/circulars.moderation.tsx

```diff
@@ -19,6 +19,12 @@ export default function () {
   return (
     <>
       <h1>Circulars Moderation</h1>
+      <Link
+        to="docs/circulars/corrections"
+        className="usa-button usa-button--outline"
+      >
+        Documentation on revision requests
```

Comment:
This could probably just be "About" , the button is fairly big. Not totally against it as is though:
![image](https://github.com/nasa-gcn/gcn.nasa.gov/assets/11023698/ba52279c-f416-41c1-87c0-f6bcd8868bfc)